docs/adrs/0016-government-cloud-comparison.md

```diff
@@ -49,6 +57,7 @@ but these are not suitable for hosting a custom application.
 standardize cloud hosting through [Haven](https://digilab.overheid.nl/projecten/haven/) for all Dutch
 municipalities. This standardization improves on integration later on.
     - Cons:
+        - It is not vanilla Kubernetes
```

Comment:
It is actually is, or did I miss something?

Reply:
ah sorry, that is my bad I assumed this :s